tests/rundb/test_httpdb.py

```diff
@@ -132,20 +132,30 @@ def cleanup():
 
 
 def server_fixture():
-    proc = None
+    process = None
     workdir = None
 
     def create(env=None):
-        nonlocal proc, workdir
+        nonlocal process, workdir
         workdir = create_workdir()
-        proc, url = start_server(workdir, env)
+        process, url = start_server(workdir, env)
         conn = HTTPRunDB(url)
         conn.connect()
         return Server(url, conn, workdir)
 
     def cleanup():
-        if proc:
-            proc.terminate()
+        if process:
+            process.terminate()
+            stdout = process.stdout.read()
+            human_readable_stdout = (
+                stdout.decode("utf-8").replace("\\n", "\n").replace("\\t", "\t")
```

Comment:
this: https://stackoverflow.com/a/24519338/1875031
or this: https://stackoverflow.com/a/37059682/1875031 (I'd go with this one personally)